docs/v3/documentation/data-formats/tlb/msg-tlb.mdx

```diff
@@ -272,7 +272,7 @@ workchain_id:int8 address:bits256  = MsgAddressInt;
 
 | Structure    | Type        | Required | Description                                                                                                      |
 |--------------|-------------|----------|------------------------------------------------------------------------------------------------------------------|
-| addr_std$10  | Constructor | Required | `$10` tag means, that in the serialization MsgAddressExt started with a `10` bits describes an external address. |
+| addr_std$10  | Constructor | Required | `$10` tag means, that in the serialization MsgAddressInt started with a `10` bits describes an internal address. |
```

Comment:
Well, `$10` tag means that the address that follows is an internal address written in std form, but in general it doesn't make much difference, it doesn't necessarily need to be fixed.